src/components/molecules/Slider/index.js

```diff
@@ -0,0 +1,85 @@
+import React, { useState } from 'react';
+import Button from '../../atoms/Button';
+import CarouselText from '../CarouselText';
+import './style.scss';
+
+const Slider = ({ title, subtitle, images }) => {
+  const slides = [
+    <img key="1" src={images && require(`../../../images/${images[2].src}`).default} alt={images && images[2].alt} className="slider__slide__img"/>,
+    <img key="2" src={images && require(`../../../images/${images[0].src}`).default} alt={images && images[0].alt} className="slider__slide__img"/>,
+    <img key="3" src={images && require(`../../../images/${images[1].src}`).default} alt={images && images[1].alt} className="slider__slide__img"/>,
+    <img key="4" src={images && require(`../../../images/${images[2].src}`).default} alt={images && images[2].alt} className="slider__slide__img"/>,
+    <img key="5" src={images && require(`../../../images/${images[0].src}`).default} alt={images && images[0].alt} className="slider__slide__img"/>,
+  ];
+  const [transform, setTransform] = useState(-100);
+  const [transition, setTransition] = useState('transitionOn');
+  const [step, setStep] = useState(1);
```

Comment:
probably use currentSlide instead of step, since it describes more clearly what this state stands for, step can be amount of items that you go trough when clicking on "next" button. So I would expect using your component by seting step to 3 I will skip 3 slides when clicking on next button

Reply:
Renamed step to currentSlide